packages/blocks/src/api/test/registration.js

```diff
@@ -168,7 +168,7 @@ describe( 'blocks', () => {
 		it( 'should reject blocks with an invalid edit function', () => {
 			const blockType = {
 					save: noop,
-					edit: 'not-a-function',
```

Comment:
We can explicitly disallow strings, which is a valid element type for React. WDYT?